protocol-units/bridge/integration-tests/tests/eth_movement.rs

```diff
@@ -0,0 +1,155 @@
+use alloy::{
+	node_bindings::Anvil,
+	primitives::{address, keccak256},
+	providers::Provider,
+};
+use bridge_integration_tests::TestHarness;
+use bridge_shared::{
+	bridge_contracts::BridgeContractInitiator,
+	types::{Amount, HashLock, InitiatorAddress, RecipientAddress, TimeLock},
+};
+use ethereum_bridge::types::EthAddress;
+
+#[tokio::test]
+async fn test_client_should_build_and_fetch_accounts() {
+	let scaffold: TestHarness = TestHarness::new_only_eth().await;
+
+	// Start Anvil with the fixed port
+	let eth_client = scaffold.eth_client().expect("Failed to get EthClient");
+	let _anvil = Anvil::new().port(eth_client.rpc_port()).spawn();
+
+	let expected_accounts = vec![
+		address!("f39fd6e51aad88f6f4ce6ab8827279cfffb92266"),
+		address!("70997970c51812dc3a010c7d01b50e0d17dc79c8"),
+		address!("3c44cdddb6a900fa2b585dd299e03d12fa4293bc"),
+		address!("90f79bf6eb2c4f870365e785982e1f101e93b906"),
+		address!("15d34aaf54267db7d7c367839aaf71a00a2c6a65"),
+		address!("9965507d1a55bcc2695c58ba16fb37d819b0a4dc"),
+		address!("976ea74026e726554db657fa54763abd0c3a0aa9"),
+		address!("14dc79964da2c08b23698b3d3cc7ca32193d9955"),
+		address!("23618e81e3f5cdf7f54c3d65f7fbc0abf5b21e8f"),
+		address!("a0ee7a142d267c1f36714e4a8f75612f20a79720"),
+	];
+
+	let provider = scaffold.eth_client.unwrap().rpc_provider().clone();
+	println!("provider: {:?}", provider);
+	let accounts = provider.get_accounts().await.expect("Failed to get accounts");
+	assert_eq!(accounts.len(), expected_accounts.len());
+
+	for (account, expected) in accounts.iter().zip(expected_accounts.iter()) {
+		assert_eq!(account, expected);
+	}
+}
+
+#[tokio::test]
+async fn test_client_should_deploy_initiator_contract() {
+	let mut harness: TestHarness = TestHarness::new_only_eth().await;
+	let anvil = Anvil::new().port(harness.rpc_port()).spawn();
+
+	// Set a funded signer
```

Comment:
It seems to me like the `// Set a funder signer` comments throughout this file aren't needed as the next line is fairly self-descriptive.